src/platform/Darwin/DnssdImpl.cpp

```diff
@@ -461,13 +461,16 @@ static CHIP_ERROR GetAddrInfo(void * context, DnssdResolveCallback callback, uin
     else
     {
         // -1 is the local only interface. In this case, we can use localhost ::1
+        /*
```

Comment:
As long as the Remove call in the callback doesn't fail, I'd vote that we just delete this code block entirely.

Reply:
I've deleted it and fixed the memory leak it would cause.